Tasks/AzureRmWebAppDeployment/Tests/L0.ts

```diff
@@ -30,7 +30,7 @@ describe('AzureRmWebAppDeployment Suite', function() {
             assert(tr.stderr.length == 0 || tr.errorIssues.length, 'should not have written to stderr');
             var expectedOut = 'Updated history to kudu'; 
             assert(tr.stdout.search(expectedOut) > 0, 'should have said: ' + expectedOut);
-            expectedOut = 'Successfully updated scmType to VSTSRM';
+            expectedOut = 'loc_mock_SuccessfullyUpdatedAzureRMWebAppConfigDetails';
```

Comment:
Where is this defined and how are you testing that the new message is getting printed to ensure that we called UpdateMetadat?